### PR TITLE
Migrating from Dubclub

### PR DESCRIPTION
**Pull request by henryk89**

### CODE REVIEW

File: src/pages/dubclub.mdx
Comment (gwkline) [inline src/pages/dubclub.mdx:1]: This doesn't really say anything about the actual migration. Can they bring their existing customers over? If yes, explain how. If not, explain that and tell why (similar to the Patreon guide)

**Review by gwkline — CHANGES_REQUESTED** [review]

**Comment by stevo2105** [comment]
#### 📝 Changed routes:


| [**/dubclub**](https://new-documentation-mdq9driab-whop.vercel.app/dubclub) |
|:-:|
|<a href="https://new-documentation-mdq9driab-whop.vercel.app/dubclub">Screenshot of /dubclub</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/uGDKaoGewQ7WOmIfDvPwsLlovhjhF-SDy7H7fMbb7qhA11j7proetp3Gr-2O_D7uktGyy-T9YswStiPd7Mzgng.png">(view full size)</a> |







Commit 68b856272f444454126f8dc5decde719080fe9bb (https://new-documentation-mdq9driab-whop.vercel.app).

**Review by gwkline — APPROVED** [review]

File: src/pages/dubclub.mdx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Migrating from Dubclub
+
+Wanting to sell Sports picks with the best tools and the lowest fees on the marketplace? With Dubclub charging 18% fees compared to Whop's 3% fees, the fees alone are more than enough reason to switch to Whop.
```
Comment (jacc) [inline src/pages/dubclub.mdx:3]: Wording here is very clunky - potentially `With Dubclub charging 18% vs. Whop's 3%, the fees alone are worth the switch`.

File: src/pages/dubclub.mdx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+## Signing up for Whop
+
+To start switching to Whop, you must signup for a business. You can follow our documentation here on how to set up your company and product page.
+
+[Getting started →](/creating-company)
+
+## Announcing Whop to your Dubclub customers
+
+Now that you have signed up for your Whop account, you must announce to your current customers that you’re changing your primary point of sale. We highly recommend you post in your Discord/platform, saying you will be changing your dashboard - you can offer a promo’code / discount off their first month on Whop for making the switch to ensure all of your customers move over.
```
Comment (jacc) [inline src/pages/dubclub.mdx:17]: i have no idea how Dubclub works but maybe make this announcement more clear - ex. card only, no PayPal currently, etc.

Comment (jacc) [inline src/pages/dubclub.mdx:17]: typo with `promo'code`

File: src/pages/dubclub.mdx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+## Announcement template
+
+```
+Hey guys, mini announcement!
+
+We’re excited to announce that we’ll be moving from Patreon to Dubclub!
+
+Whop is a platform similar to Dubclub. However, they have cheaper fees and many more features - you can now leave written reviews, become an affiliate & get paid for referring friends!
```
Comment (jacc) [inline src/pages/dubclub.mdx:26]: good highlight of the consumer facing features but needs better wording - ex: `Whop is similar [...] they also make it easier to write reviews and reward YOU with a native affiliate program`

File: src/pages/dubclub.mdx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+We will also be featured on their marketplace, which we are super excited about, and their infrastructure enables us to sell more than just Discord roles - including courses, software, and more!
```
Comment (jacc) [inline src/pages/dubclub.mdx:28]: these should be two different points - maybe include featured in the point above?

drop the line about infrastructure and make it more "customer facing" - ex, "whop also will allow us to include more than just Discord access - we can also include courses, software, and more with your subscription on their platform!"

**Review by jacc — APPROVED** [review]

**Pull request merged**